sematic/db/migrations/20230526141354_add_run_command_and_config_to_resolutions.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+ALTER TABLE resolutions ADD COLUMN run_command TEXT;
+ALTER TABLE resolutions ADD COLUMN build_config TEXT;
```

Comment:
Should we store it as a JSON payload? Right now it's raw yaml right?

Reply:
Yes, it is `YAML`. Is there any transformation you expect to do in order to store it as `JSON` instead of `TEXT`? Is there any value in breaking the config into parts at this point?

If we decide to interpret the config for the future (re)run from the Dashboard using UI inputs, we can impose whatever parsing / transformation / interpretation of the config we decide we need (or not) during the design of that feature, as the current format is raw, and amenable to any transformations.